### ISSUE TITLE
Use API filtering in rocky/rocky/views/finding_list.py FindingListView

### ISSUE THREAD

**Issue by underdarknl** [minvws/nl-kat-coordination#573]
Currently we are filtering the findings in python time instead of requesting just the limited / filtered set from the backend api.
This is not very performant, and should be improved by asking the Octopoes API for the filtered set of findings as requested by the user.

**Comment by ammar92** [comment]
Somewhat related to https://github.com/minvws/nl-kat-coordination/issues/396?

**Comment by underdarknl** [comment]
Yup, although we also don't use all already present filtering/limiting api's.

**Comment by underdarknl** [comment]
Maybe we have by now implemented all these filters correctly?
